apps/correct-mistakes/fields/about-error.js

```diff
@@ -28,16 +30,19 @@ module.exports = {
     }
   },
   'date-of-birth-error-checkbox': {
+    mixin: 'checkbox',
     toggle: 'date-of-birth-error-container'
   },
   'date-of-birth-error': date('date-of-birth-error', {
+    mixin: 'input-date',
     validate: ['required', 'before'],
-    dependent: {
+    validationLink: {
```

Comment:
what is validationLink?

Reply:
it is similar to 'dependent' but used when there are multiple dependent fields. The input-date mixin does not work with the 'dependent' property as it has multiple dependent fields in one field so the validationLink property is used instead - https://github.com/UKHomeOfficeForms/hof/blob/cd4be044e5b4c5c44f162faccf8b5603f8b7dda4/controller/validation/index.js#L67

Reply:
I think I spoke to @SarahJaneLuff about this, can't remember how long ago but great to see it's being used.  Nice one @Rhodine-orleans-lindsay and Sarah